src/bionics.cpp

```diff
@@ -272,7 +288,7 @@ void player::power_bionics()
     const int tabs_start = 1;
     const int tab_step = 3;
 
-    while(true) {
+    for (;;) {
```

Comment:
Apologies in advance for the rant, but:

I still don't get why a (C++) compiler would warn about `while(true)`, but not about a `for(;;)`.

I get why it would warn on `const bool x=true;while(x)` or similar, but the intend of `while(true)` is completely clear. And arguing that it could be an accident, well, `for(;;)` could be as well, maybe `for(;y;)` was meant?

`for( ; ; )` looks like "I write this as placeholder and fill in the actual code later."

----

The abode does not apply to C.